server/src/test/scala/RoutesTests.scala

```diff
@@ -0,0 +1,16 @@
+package server
+
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import org.scalatest.FunSuite
+
+class RoutesTests extends FunSuite with ScalatestRouteTest {
+  test("Static routes") {
+    Get("/") ~> Routes() ~> check {
+      assert(responseAs[String].contains("doctype"))
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The string literal "doctype" appears 2 times in the file.](https://www.codacy.com/app/olivier-blanvillain/19x19-net/file/2596283663/issues/source?bid=3301982&fileBranchId=3306761#l9)